### ISSUE TITLE
Speaker's List Broward County 

### ISSUE THREAD

**Issue by RobDavis** [CodeForFtL/project-ideas#4]
## Short description of the idea (140 character tweet):

Find a special local speaker for your tech-related opportunities! 

## What is your project concept or idea? What challenge or opportunity will it solve?

This application was built to maintain a list of individuals interested in speaking at technology-related events in Broward County.

### Do you have photos of them? Or quotes?

Residents, local tech clubs, schools, organizations (i.e. Business, Conferences, etc.) that are trying to find a local tech speaker. Personal experience drove this project, as we've tried to find female speakers in Broward and had limited success. 

## Issue area(s) relevant to your project idea:

- Children & Youth
- Economic Opportunity & Development
- Education

## What is the current state of your project idea?

- I have an idea or **concept** of a solution, but have not yet prototyped or tested it

### Do you have any links to news articles, text, mockups, code, or data?

The Project is online, but needs updates. 
Live: http://codeforftl.org/speakers-list/
Code: https://github.com/CodeForFtL/speakers-list

### Are you representing a group in submitting this project idea? 

Inspired by http://wespeaktoo.org/, built by @kklumpp13

**Comment by rachdaup93** [comment]
I am interested in contributing to this project.

**Comment by yfrigi** [comment]
I am interested.  I can speak on the Bail Project, Social Justice and Cybersecurity in plain English so anyone can understand.

**Comment by qtrandev** [comment]
Can anyone interested in being added to this list message me or post here the following information?

Name/Languages/Topics/Contact
Example: Speaker 1/React,Angular,Node/Front End Development, Responsive Design, Accessibility/speaker1@test.com

I can update http://codeforftl.org/speakers-list/ if needed.

**Comment by ryanvgates** [comment]
@kklumpp13 @yfrigi @rachdaup93 @qtrandev how's this project going? What would you want from new volunteers to help move the project forward?

**Comment by kklumpp13** [comment]
@ryanvgates I was just talking to a co-worker who is a React developer yesterday about working on this project together. The project currently isn't responsive or accessible, but that should be pretty straightforward to fix. I do have a question though. Back when I first set up the project, there was a discussion about using Firebase so we didn't have to manually update the speaker list. Are there any other recommendations now?
